test/e2e/utils.go

```diff
@@ -261,6 +262,26 @@ func currentState(namespace string, node string, currentStateYaml *nmstatev1alph
 	}, ReadTimeout, ReadInterval)
 }
 
+// TODO log actual condition when fails
```

Comment:
Please tackle this. In case we don't get expected condition, we should print the list of actual conditions found in the State.